micropub.php

```diff
@@ -593,15 +593,31 @@ public static function generate_post_content( $args ) {
 			'like-of'     => 'Likes',
 			'repost-of'   => 'Reposted',
 			'in-reply-to' => 'In reply to',
+			'bookmark-of' => 'Bookmarked',
 		);
 
 		// interactions
 		foreach ( array_keys( $verbs ) as $prop ) {
-			$val = $props[ $prop ][0];
+			if ( wp_is_numeric_array( $props[ $prop ] ) ) {
+				$val = $props[ $prop ][0];
+			} else {
+				$val = $props[ $prop ];
+			}
 			if ( $val ) {
-				$lines[] = '<p>' . $verbs[ $prop ] .
-				  ' <a class="u-' . $prop . '" href="' .
-				  $val . '">' . $val . '</a>.</p>';
+				$l = '<p>' . $verbs[ $prop ] .
+				' <a class="u-' . $prop . '" href="';
+				if ( is_string( $val ) ) {
+					$l .= $val . '">' . $val;
+				}
+				if ( is_array( $val ) ) {
```

Comment:
mind adding a comment with a link to the nested properties part of the spec here? https://micropub.net/draft/#nested-microformats-objects